manifests/0000_90_kube-controller-manager-operator_05_cluster-policy-controller-role.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-policy-controller-leader-lock
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  resourceNames: ["cluster-policy-controller"]
```

Comment:
this won't allow you to create.  You probably already have permissions, but if not, just grant yourself access to all the resource names.

Reply:
oh, that's it! ok